lib/filesystem/FileStream.cpp

```diff
@@ -0,0 +1,148 @@
+#include "StdInc.h"
+#include "FileStream.h"
+#include "../minizip/unzip.h"
```

Comment:
OK. Also it should be
```cpp
#ifdef USE_SYSTEM_MINIZIP
#include <minizip/unzip.h>
#else
#include "../minizip/unzip.h"
#endif
```

In debian linux at least there is no minizip in vcmi#ifdef USE_SYSTEM_MINIZIP
#include <minizip/unzip.h>
#else
#include "../minizip/unzip.h"
#endif
```